uber/templates/macros.html

```diff
@@ -82,6 +82,36 @@
 {%- endmacro %}
 
 
+{% macro form_group_scaffolding(
+    label=False,
+    help=False,
+    is_required=False,
+    size=6,
+    form_class='',
+    static_wrap=False) -%}
```

Comment:
Whether or not the caller's content should be static wrapped should probably be determined by the calling function. The point of this macro is to wrap a form-control inside a form-group. The form-control itself should determine whether or not it's static.